packages/react-sdk/src/components/Layout/Layout.tsx

```diff
@@ -77,7 +79,11 @@ export function Layout({ height = '100vh' }: LayoutProps) {
     <SlidesProvider>
       <GuidedTour disabled={isViewingPresentation} />
 
-      <Stack height={height} direction="row" bgcolor="background.paper">
+      <Stack
+        height={!isFullscreenMode ? height : '100vh'}
```

Comment:
This ensures we dont get a black bar under the widget if its smaller than 100vh normally.